README.md

```diff
@@ -25,3 +25,5 @@ Additional considerations:
     -   What needs to live in a persistence layer?
 -   Is there some state we need to initialize?
 -   Ask: should any of this work be abstracted into functions? (i.e., is the work complicated? can it be reused?)
+
+[planning doc](https://whimsical.com/rps-hyrule-CGcj6rbTtVK4vYcQe59zjA)
```

Comment:
I dont have access to the page, in the future I would just put a screenshot in the repo.